### DISCUSSION TITLE
Welcome to programming-guidelines Discussions!

### DISCUSSION THREAD

**Discussion by guettli** [guettli/programming-guidelines#11] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by emmysteven** [comment]
Hello, I'm so glad that you have activated **discussions**

**Comment by richardgourley** [comment]
Hi! Great tips.
I'm a Python and django developer, so combining these tips with your [https://github.com/guettli/django-tips](url) repo has changed how I approach django projects and general development.

Just an idea:
I think the sections here about databases could be really useful if added to the django-tips repo, even if briefly as DB tips for Django.  Any thoughts?

These sections in particluar:

```Avoid nullable Foreign Keys```

```Avoid nullable boolean columns```

```Avoid nullable characters columns```
